ddtrace/profiling/profiler.py

```diff
@@ -55,6 +57,15 @@ def __repr__(self):
 ProfilerStatus.RUNNING = ProfilerStatus("running")
 
 
+def gevent_patch_all(event):
+    if "ddtrace.profiling.auto" in sys.modules:
+        warnings.warn(
+            "Starting the profiler before using gevent monkey patching is not supported "
+            "and is likely to break the application.",
+            RuntimeWarning,
```

Comment:
Is it worth mentioning the `DD_GEVENT_PATCH_ALL` feature to help remedy the situation? https://ddtrace.readthedocs.io/en/latest/integrations.html#gevent

Reply:
Good idea!